### ISSUE TITLE
ambiguous use of 'init(string:)

### ISSUE THREAD

**Issue by vhosune** [delba/TextAttributes#9]
On Xcode 7.3 I get this error, "ambiguous use of 'init(string:)", I tried to force the namespace Foundation but it's still ambiguous

```
var s = Foundation.NSMutableAttributedString(string: text)
                    ^
Foundation.NSMutableAttributedString:14:12: note: found this candidate
    public init(string str: String)
           ^
TextAttributes.NSAttributedString:19:24: note: found this candidate
    public convenience init(string: String, attributes: TextAttributes.TextAttributes? = default)
```

**Comment by delba** [comment]
Hi @vhosune , thanks for the issue!

It seems that @pixyzehn has solved that in #7 . Can you confirm?
I'm gonna release a new version including his commit later today.

**Comment by delba** [comment]
@vhosune I just push `v0.2.5` with @pixyzehn's fix.

Enjoy 😃
